demisto_sdk/commands/common/constants.py

```diff
@@ -595,11 +595,13 @@ class FileType(str, Enum):
 DEPRECATED_DESC_REGEX = r"Deprecated\.\s*(.*?Use .*? instead\.*?)"
 DEPRECATED_NO_REPLACE_DESC_REGEX = r"Deprecated\.\s*(.*?No available replacement\.*?)"
 PACK_NAME_DEPRECATED_REGEX = r".* \(Deprecated\)"
+DEPRECATED_COMMAND_REGEX = r"Command \*\*\*.*?\*\*\* is deprecated. Use .*? instead."
```

Comment:
What happens when there's no alternative?

Reply:
It fails like it was before in the bug